src/style/general.st.css

```diff
@@ -0,0 +1,45 @@
+:global(body) {
+  margin: 0;
+}
+
+
```

Comment:
remove newline

Reply:
you liked but didn't remove

Reply:
you are right!

removed in a different branch